3rd-party/nccl/neuron/include/nccl/err.h

```diff
@@ -0,0 +1 @@
+error.h
```

Comment:
?

Reply:
nccl does not ship error.h, they ship err.h:  https://github.com/NVIDIA/nccl/blob/master/ext-net/example/nccl/err.h

neuron does not ship err.h, they ship error.h: https://github.com/aws/aws-ofi-nccl/tree/master/include/nccl-headers/neuron

this is how github displays symlinks.

Reply:
Ah, that's... lame of github.